src/main/java/com/google/sps/data/IAMBindingDatabaseEntry.java

```diff
@@ -0,0 +1,21 @@
+package com.google.sps.data;
+
+/** Represents a single row of the IAM Bindings table. */
+public class IAMBindingDatabaseEntry {
+
+  public String projectId;
```

Comment:
Is this meant to be mutable?

If not, please consider @AutoValue or at least wrapping controlling the visibility of these members and wrapping them in getters. Here and throughout this PR.

Reply:
I've changed all the classes to @AutoValue classes so they're all generated now!